scripts/gen-api.sh

```diff
@@ -19,17 +18,19 @@ else
   exit 1
 fi
 
+SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
+ROOT_DIR="$SCRIPT_DIR/.."
+
```

Comment:
```suggestion
cd "$ROOT_DIR"

```
実行時のカレントディレクトリがプロジェクト直下でない場合に、npxが実行できなかったり(別の場所にインストールしようとしてしまう)、またはopenapitools.jsonが変な場所に生成されてしまいます